src/MarketingPageAcceptanceTests.TestData/Capabilities/EpicDto.cs

```diff
@@ -13,7 +13,7 @@ public sealed class EpicDto
 
         internal IEnumerable<EpicDto> GetAllByIdPrefix(string connectionString, string id)
         {
-            var query = Queries.GetAllEpicsByCapabilityPrefix;
+            var query = "SELECT * FROM Epic WHERE Id LIKE @ID";
```

Comment:
SHould this be "like @ID%" for a prefix?

Reply:
Added the prefix '%' into the param as putting it in after the variable name breaks the sql query